src/endpoints/Endpoint.js

```diff
@@ -54,10 +50,10 @@ class Endpoint {
    * @param {string} name The name of this endpoint. This is used as key in the state and as Payload name. The Payload name is name + 'Paylaod'
    * @param {string} url The url with params (params are used like this: https://cms.integreat-app.de/{location}/{language})
    * @param {function} jsonToAny Transforms the json input to a result
-   * @param {mapStateToOptionsCallback} mapStateToUrlParams Maps the state to the url params which are needed in the Fetcher component
+   * @param {mapStateToOptionsCallback} mapStateToOptions Maps the state to the url params which are needed in the Fetcher component
    * @param shouldRefetch Takes the current and the next props and should return whether we should refetch
    */
-  constructor ({name, url, jsonToAny, mapStateToOptions = DUMMY, shouldRefetch = () => false}) {
```

Comment:
you could rename jsonToAny to transformJsonToResult or similar? 🤓

Reply:
Sure, this is still in progress, just wanted to share my ideas